### PR TITLE
py-klein: new port

### PR DESCRIPTION
**Pull request by JacksonIsaac**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.13.6 17G65
Xcode 9.3 9E145

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?

- [x] checked your Portfile with `port lint`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-klein/Portfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+distname            ${python.rootname}-${version}
+
+python.versions     36 37
+
+if {${name} ne ${subport}} {
+    depends_lib-append  \
+        port:py${python.version}-setuptools \
+        port:py${python.version}-twisted \
+        port:py${python.version}-idna \
+        port:py${python.version}-werkzeug
```
Comment (mf2k) [inline python/py-klein/Portfile]: It's minor, but please sort the dependencies.

Comment (JacksonIsaac) [reply python/py-klein/Portfile]: Thanks for pointing this out. I was actually wondering how the list of dependencies should be ordered.

File: python/py-klein/Portfile [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           select 1.0
```
Comment (reneeotten) [inline python/py-klein/Portfile]: it looks like you're not actually using the select PortGroup or...?

Comment (pmetzger) [reply python/py-klein/Portfile]: If that's the case, perhaps this should be fixed before a merge.

Comment (JacksonIsaac) [reply python/py-klein/Portfile]: oh yeah, maybe I took a Portfile example where *_select subport also exists. I'll fix this for all the python packages that were submitted as PR.

Also what is the best way to push changes ? force push to branch as single commit using commit --amend ?

Comment (pmetzger) [reply python/py-klein/Portfile]: Force push, yah.

**Comment by pmetzger** [comment]
Can you squash all the commits into one and force push?

**Comment by pmetzger** [comment]
Merged! Thanks, @JacksonIsaac!

**Pull request merged**